app/modules/asset_groups/resources.py

```diff
@@ -347,11 +347,16 @@ def delete(self, asset_group):
         """
         Delete an Asset_group by ID.
         """
+        _, asset_group_id = asset_group
         asset_group = self._get_asset_group_with_428(asset_group)
 
         if asset_group is not None:
             asset_group.delete()
 
+        from app.modules.asset_groups.tasks import delete_remote
+
+        delete_remote.delay(str(asset_group_id))
```

Comment:
It would make a lot more sense to my mind for the delete_remote task call to be with in the asset_group delete method. 
That would also mean that the tests wouldn't need to handle this gitlab specific case separately.

Reply:
This allows us to delete the asset group if it's not in the database but is on gitlab.... Now that I think about it, may be `_get_asset_group_with_428` will create the asset group if it exists on gitlab...?

Reply:
That's exactly what it does. 
I understand that the potential future goal is that we may want to delete the images in houston and keep them on Gitlab only but we would not delete the asset groups in this case. 
The only time when we want to delete the asset groups locally, we really want to delete them, gitlab and all. 
The  `_get_asset_group_with_428` is handling when this has gone wrong which it shouldn't in the real world ...... but is extremely useful for testing.

Reply:
What's the thing to do here?  I added `delete_remote` here because I wanted to delete the asset group on gitlab even if the asset group doesn't exist in the database.

We can instead add `delete_remote` to `asset_group.delete()` and delete the gitlab project only if the asset group exists in the database.

Reply:
That makes sense, I'll make the changes :+1: